students/AurelPerianu/Lesson7/html_render.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+
+class Element:
+    """Base class to run functions"""
+    tag = "html"
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+        self.atrbs = kwargs
+
+    def append(self, content_str):
+        """append content to the element class object"""
+        self.content.append(content_str)
+
+    def render(self, resp, cur_ind=0):
+        """general render for html file"""
+        resp.write(f'{cur_ind*self.indent}<{self.tag}{self.extra_attrib()}>\n')
+        for item in self.content:
+            try:
+                item.render(resp, cur_ind+1)
+            except AttributeError:
+                resp.write((cur_ind + 1) * self.indent + str(item) + '\n')
+        resp.write(f'{cur_ind * self.indent}</{self.tag}>\n')
+
+    def extra_attrib(self):
+        atb = ''
+        if self.atrbs:
+            for key, value in self.atrbs.items():
+                atb += f' {key}="{value}";'
+        return str(atb)
+
+
+class OneLineTag(Element):
+
+    def render(self, resp, cur_ind=0):
+        ax = f'{(cur_ind) * self.indent}<{self.tag}>'
+        ay = f'{self.content[0]}{self.extra_attrib()}</{self.tag}>\n'
```

Comment:
It's a best practice to use meaningful variable names for these kinds of assignments and makes your code more readable